.github/PULL_REQUEST_TEMPLATE/new_feature.md

```diff
@@ -0,0 +1,3 @@
+## PR checklist:
+- [ ] `docs/` updated
+- [ ] tests added
```

Comment:
I think it would be great to include examples, especially where a new feature has been added of the functionality behind it has changed.

Perhaps an updated `main.tf` to show the change quickly for reference but also stop that from getting stale.

Reply:
Yeah I like that. I'll add another item.

I also wonder if we could make better use of the docs that end up on the terraform registry to include examples there. But I'll leave that for another time I think

Reply:
```suggestion
- [ ] `docs/` updated
- [ ] tests added
- [ ] an example added to `examples/` (useful to demo new field/resource
```